flexmeasures/cli/data_edit.py

```diff
@@ -302,6 +308,114 @@ def transfer_ownership_recursive(asset: Asset, account: Account):
     db.session.commit()
 
 
+@fm_edit_data.command("transfer-parenthood")
+@with_appcontext
+@click.option(
+    "--asset",
+    "asset",
+    type=AssetIdField(),
+    required=False,
+    help="Change/set the parent of this asset. Follow up with the asset's ID.",
+)
+@click.option(
+    "--new-parent",
+    "new_parent",
+    type=AssetIdField(),
+    required=False,
```

Comment:
```suggestion
    required=True,
```

I believe we require this.